test/e2e/common/runtime.go

```diff
@@ -265,7 +265,7 @@ while true; do sleep 1; done
 
 		ginkgo.Context("when running a container with a new image", func() {
 
-			// Images used for ConformanceContainer are not added into NodeImageWhiteList, because this test is
+			// Images used for ConformanceContainer are not added into NodeImageAllowList, because this test is
 			// testing image pulling, these images don't need to be prepulled. The ImagePullPolicy
 			// is v1.PullAlways, so it won't be blocked by framework image white list check.
```

Comment:
We should also update this comment to follow the name change in https://github.com/kubernetes/kubernetes/pull/92318
```suggestion
			// is v1.PullAlways, so it won't be blocked by framework image pre-pull list check.
```